src/components/Answers/Answers.tsx

```diff
@@ -8,33 +8,38 @@ interface Iprops{
     name: string
     population: number
     capital: string
+    subregion: string
+    
     borders: string[]
+    flag: string
   } 
 }
 
 const Answers: React.FC<Iprops> = ({ currentCountry, questions, guess }) => {
 
+  const [answer, setAnswer] = useState<string>('')
```

Comment:
great idea to add the answer as a state property in order to easily make the comparison!